src/modules/BaseLayout/BaseLayout.module.css

```diff
@@ -0,0 +1,12 @@
+@import '_colors.css';
+
+.app-layout {
```

Comment:
`.layout`? Here and in  the same case too

Reply:
done